From 1fa8670b27f5a2b2a790df2ee19817f368c07084 Mon Sep 17 00:00:00 2001
From: Alfred Neumayer <dev.beidl@gmail.com>
Date: Thu, 30 Sep 2021 20:22:29 +0200
Subject: [PATCH 14/35] vayu_user_defconfig: Enable config switches for
 WireGuard

This enables the in-tree WireGuard driver to be used
using 'sudo ip link add test_wg0 type wireguard'.
---
 arch/arm64/configs/vayu_user_defconfig | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/configs/vayu_user_defconfig b/arch/arm64/configs/vayu_user_defconfig
index 83f3fa5fd3eb..f7fecdf4a0d7 100644
--- a/arch/arm64/configs/vayu_user_defconfig
+++ b/arch/arm64/configs/vayu_user_defconfig
@@ -864,3 +864,7 @@ CONFIG_CGROUP_SCHEDTUNE=n
 CONFIG_USB_CONFIGFS_RMNET_BAM=y
 
 CONFIG_PM_AUTOSLEEP=y
+
+CONFIG_NET_UDP_TUNNEL=y
+CONFIG_CRYPTO_ALGAPI=y
+CONFIG_WIREGUARD=y
-- 
2.39.2

